civihr_default_theme/template.php

```diff
@@ -14,6 +14,169 @@
 require_once dirname(__FILE__) . '/includes/view.inc';
 
 /**
+ * Implements template_preprocess_views_view_table().
+ */
+function civihr_default_theme_preprocess_views_view_table(&$variables) {
+  $variables['classes_array'] = array_diff($variables['classes_array'], array('table-striped', 'table-bordered'));
+}
+
+/**
+ * Implements theme_table().
+ */
+function civihr_default_theme_table($variables) {
```

Comment:
Yes it was copied from radix. I have added a comment specifying which part has been changed. Otherwise I dont think its a good idea to split the functions as its copied from radix, so incase radix function changes we can easily copy that back to here .